llvm/test/CodeGen/X86/inline-asm-memop.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: llc -mtriple=x86_64-unknown-linux-gnu < %s
+
+; A bug in X86DAGToDAGISel::matchAddressRecursively create a zext SDValue which
+; is quickly replaced by other SDValue but already pushed into vector for later
+; calling for SelectionDAGISel::Select_INLINEASM getNode builder, see issue
+; 82431 for more infomation.
+
+define void @d(i8 %call, ptr %b) {
```

Comment:
(style) rename this @PR82431

Reply:
Excuse me, which name should I pick : @PR82881 or mention the related issue in another style ?

Reply:
Historically, PR is an abbreviation for "Problem Report", so @PR82431 would be correct. See https://llvm.org/docs/TestingGuide.html#other-features

Reply:
Apparently, https://llvm.org/PR82431 opens the wrong issue.

@RKSimon I've seen other people use GHxxxxx syntax. Could it be a better choice now?

Reply:
The PR#### number should be the bug issue report, not the pull request. PR doesn't mean Pull Request it means Problem Report.